recipes/restinio/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.12)
```

```suggestion
cmake_minimum_required(VERSION 3.1.3)
```
TARGETS and set_target_properties require CMake 3